pkg/component/worker/kubelet.go

```diff
@@ -154,7 +154,6 @@ func (k *Kubelet) Run(ctx context.Context) error {
 		args["--cni-bin-dir"] = "C:\\k\\cni"
 		args["--cni-conf-dir"] = "C:\\k\\cni\\config"
 		args["--hostname-override"] = node
-		args["--cluster-domain"] = "cluster.local"
```

Comment:
I'm not sure why this was explicitly set only for windows as it's set in the kubelet config anyway (the command line argument is deprecated)

Reply:
Probably some left-over from early days... :)